### DISCUSSION TITLE
Data refresh panel: maybe add the differente refresh buttons at the top

### DISCUSSION THREAD

**Discussion by zenzeinet** [TabularEditor/TabularEditor3#1119] [Ideas]
TE3: Data refresh panel: maybe add the differente refresh buttons at the top? (or at least Automatic and Calculate)

**Comment by otykier** [comment]
Could you clarify what you mean by this? The Data Refresh panel doesn't currently provide a way to perform a refresh, as that is typically triggered at the object level (table, partition, etc.). However, I guess it could make sense to provide shortcuts within the Data Refresh panel, for queueing various types of refreshes at the model level?

**Comment by zenzeinet** [comment]
I would say a "calculate" (model) along with an "automatic" (model) are probably the two most used refresh types - would be nice to have them easily reachable from the Data Refresh Panel. I would avoid a "full" button as it can be dangerous if it cannot easily be canceled (as per the other open issues/request) ((or add a confirmation message just in case))
